typescript/infra/scripts/deploy.ts

```diff
@@ -194,6 +228,24 @@ async function main() {
       undefined,
       contractVerifier,
     );
+  } else if (module === Modules.HOOK) {
+    const ismFactory = HyperlaneIsmFactory.fromAddressesMap(
+      getAddresses(environment, Modules.PROXY_FACTORY),
+      multiProvider,
+    );
+    deployer = new HyperlaneHookDeployer(
+      multiProvider,
+      hyperlaneEnvironments[env],
+      ismFactory,
+    );
+    // Config is intended to be changed for ad-hoc use cases:
+    config = {
+      ethereum: {
+        ...(coreConfig.ethereum.defaultHook as FallbackRoutingHookConfig)
+          .domains.ancient8,
+        owner: DEPLOYER,
+      },
+    };
```

Comment:
interesting, I kind of expected this to be `core.defaultHook` but thats also a bit awkward

Reply:
honestly I don't totally remember why we did it like this but I feel like we had a good reason, something to do with contract ownership or keeping the deployed hook as simple as possible